tools/apidiff/report/package.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package report
```

Comment:
Package `report` has been refactored to `tools/internal/report`, please use that instead

Reply:
fixed